components/_twig-components/functions/bem.function.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
I wasn't going to commit it, but once it gets added with the install script, it gets added to the repo. And I wasn't sure whether gitignoring it was a good idea either. What do you think?